uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/APIWrapper/DefaultLimitRemoverTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
+
+/**
+ * Class CRM_HRCore_APIWrapper_DefaultLimitRemoverTest
+ *
+ * @group headless
+ */
+class CRM_HRCore_APIWrapper_DefaultLimitRemoverTest extends \PHPUnit_Framework_TestCase implements HeadlessInterface, TransactionalInterface {
+
+  private $defaultLimitRemoverObj;
```

Comment:
Cmon, we have annotations. There's no need to suffix the variable with Obj to indicate this is an object